examples/humble/go/views/app.go

```diff
@@ -107,8 +107,11 @@ func (v *App) CreateTodo(ev dom.Event) {
 	if !ok {
 		panic("Could not convert event target to dom.HTMLInputElement")
 	}
-	v.Todos.AddTodo(strings.TrimSpace(input.Value))
-	document.QuerySelector(".new-todo").(dom.HTMLElement).Focus()
+	title := strings.TrimSpace(input.Value)
+	if title != "" {
```

Comment:
Honestly, I like it the better the way it is. IMHO it's a little more readable than nesting it in the if statement.